DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RootObjectBuilder.cs

```diff
@@ -42,26 +42,87 @@ RevitConversionContextStack contextStack
       throw new InvalidOperationException("No objects were found. Please update your send filter!");
     }
 
-    Base commitObject = new();
+    var doc = _contextStack.Current.Document.Document; // POC: Document.Document is funny
+    var rootObject = new Collection() { name = doc.PathName.Split('\\').Reverse().First().Split('.').First() };
```

Comment:
readability I think Last() is clearer and given split returns an array it is also faster, unless I am misreading what this is doing